### DISCUSSION TITLE
Snapcraft Installation docs are missing

### DISCUSSION THREAD

**Discussion by akamfoad** [starship/starship#2225] [Q&A]
I've just found that Starship is available as a snap package as of  **Dec 10, 2019** where #707 is merged, but I couldn't find any docs from the [Installation docs](https://starship.rs/guide/#%F0%9F%9A%80-installation "https://starship.rs/guide/#%F0%9F%9A%80-installation").
And not in the [README.md](https://github.com/starship/starship/blob/master/README.md "https://github.com/starship/starship/blob/master/README.md"), and it contains this line: 
> **Note**: due to the proliferation of different platforms, only a subset of supported platforms are
shown below. Can't see yours? Have a look at the [extra platform instructions](https://starship.rs/installing/).

Even that link doesn't contain any docs for snapcraft, is there any reason for that? if not then we can make it available in both sides so people can easily find their way to get it to work.

**Comment by davidkna** [comment]
The Advanced Installation docs where only added recently, which is why it only contains instructions for the nix package manager so far.

**Reply by akamfoad** [reply]
Sounds good, we might add Snapcraft's also. Snap badge for Starship will be cool as well. I will open an issue and make a pull request when I finish my college exams :D
